test/export/ValueSetExporter.test.ts

```diff
@@ -0,0 +1,229 @@
+  FshCode,
+  VsProperty,
+  VsOperator
+} from '../../src/fshtypes';
+import { loggerSpy } from '../testhelpers/loggerSpy';
+
+describe('ValueSetExporter', () => {
+  let defs: FHIRDefinitions;
+  let doc: FSHDocument;
+  let input: FSHTank;
+  let exporter: ValueSetExporter;
+
+  beforeAll(() => {
+    defs = new FHIRDefinitions();
+    loadFromPath(
+      path.join(__dirname, '..', 'testhelpers', 'testdefs', 'package'),
+      'testPackage',
+      defs
+    );
+  });
+
+  beforeEach(() => {
+    doc = new FSHDocument('fileName');
+    input = new FSHTank([doc], { name: 'test', version: '0.0.1', canonical: 'http://example.com' });
+    exporter = new ValueSetExporter(defs, input);
+  });
+
+  it('should output empty results with empty input', () => {
+    const exported = exporter.export();
+    expect(exported).toEqual([]);
+  });
+
+  it('should export a single value set', () => {
+    const valueSet = new FshValueSet('BreakfastVS');
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].name).toBe('BreakfastVS');
+    expect(exported[0].id).toBe('BreakfastVS');
+    expect(exported[0].title).toBeUndefined();
+    expect(exported[0].description).toBeUndefined();
+    expect(exported[0].url).toBe('http://example.com/ValueSet/BreakfastVS');
+  });
+
+  it('should export multiple value sets', () => {
+    const breakfast = new FshValueSet('BreakfastVS');
+    const lunch = new FshValueSet('LunchVS');
+    doc.valueSets.set(breakfast.name, breakfast);
+    doc.valueSets.set(lunch.name, lunch);
+    const exported = exporter.export();
+    expect(exported.length).toBe(2);
+  });
+
+  it('should export a value set with additional metadata', () => {
+    const valueSet = new FshValueSet('BreakfastVS');
+    valueSet.title = 'Breakfast Values';
+    valueSet.description = 'A value set for breakfast items';
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].name).toBe('BreakfastVS');
+    expect(exported[0].id).toBe('BreakfastVS');
+    expect(exported[0].title).toBe('Breakfast Values');
+    expect(exported[0].description).toBe('A value set for breakfast items');
+    expect(exported[0].url).toBe('http://example.com/ValueSet/BreakfastVS');
+  });
+
+  it('should export each value set once, even if export is called more than once', () => {
+    const breakfast = new FshValueSet('BreakfastVS');
+    doc.valueSets.set(breakfast.name, breakfast);
+    let exported = exporter.export();
+    expect(exported.length).toBe(1);
+    exported = exporter.export();
+    expect(exported.length).toBe(1);
+  });
+
+  it('should export a value set that includes a component from a system', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { system: 'FOOD' };
```

Comment:
Not a real big deal, but technically the string `FOOD` is not really a valid system identifier.  This test would be more realistic if it was something like: `http://food.org/food`.